controllers/groupController.js

```diff
@@ -0,0 +1,38 @@
+const groupModel = require("../models/groupModel");
+const { isValidString, isEmptyString } = require("../utils/validation");
+
+const edit = async (req, res) => {
+  if (!isValidString(req.params.groupId) || isEmptyString(req.params.groupId)) {
+    return res.status(400).send({ message: "[InvalidGroupId] Error occured" });
+  }
+  if (!isValidString(req.body.groupOwnerUid) || isEmptyString(req.body.groupOwnerUid)) {
+    return res.status(400).send({ message: "[InvalidGroupOwnerUid] Error occured" });
+  }
+  if (!isValidString(req.body.groupNewName) || isEmptyString(req.body.groupNewName)) {
+    return res.status(400).send({ message: "[InvalidGroupNewName] Error occured" });
+  }
+
+  const hasGroupName =
+    (await groupModel.findOne({
+      ownerUid: req.body.groupOwnerUid,
+      name: req.body.groupNewName,
+    })) !== null;
+  if (hasGroupName) {
+    return res.status(400).send({ message: "[ExistedGroupName] Error occured" });
+  }
+
+  try {
+    const { groupId } = req.params;
+    const { groupNewName } = req.body;
```

Comment:
```suggestion
    const groupId = req.params.groupId;
    const groupNewName = req.body.groupNewName;
```
구조분해할당할 데이터가 하나뿐이며 추후에 추가될 데이터가 없을 것으로 판단되어 변수에 바로 할당해줘도 될 것 같아요!

Reply:
그룹 이름 외 변경이 필요할 것을 고려했는데 당장은 필요 없겠다 생각이 듭니다! 수정하고 추후 필요시 대응하면 좋겠습니다 :)